spec/features/user_adds_a_new_item_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+feature 'User Adds A New Item' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.